coverart_redirect/request.py

```diff
@@ -346,7 +350,8 @@ def handle(self, request):
             return self.handle_index()
         if entity == 'robots.txt':
             return self.handle_robots()
-
+        if entity == 'api':  
+            return self.handle_api(request)
         self.validate_entity(entity)
```

Comment:
I think there should still be an empty line here, but I'll leave it to @gentlecat whether it's good enough as‐is. :)